pyecharts/charts/basic_charts/polar.py

```diff
@@ -79,4 +80,34 @@ def add(
                     "itemStyle": itemstyle_opts,
                 }
             )
+        if label_opts.get("show"):
+            self.options.get("series").append(

```

Comment:
no unit test for this so this code change become a problem for future maintenance.

Reply:
Will perfect it, this pr is not ready for merged